src/use-cases/tag-use-case.ts

```diff
@@ -66,7 +66,7 @@ export class JiraTagUseCase implements TagUseCase {
         flatMap((extractTicketsOutput) => {
           const tag = `${input.tag}${input.jiraTagSuffix}`;
           return this.createVersionUseCase
-            .execute(new CreateVersionUseCaseInput(input.project, tag))
+            .execute(new CreateVersionUseCaseInput([input.project], tag))
```

Comment:
Not sure if this will work 🤔 
`input.project` is either a `string` (as defined in the type [here](https://github.com/leonardowf/baroneza/blob/f80f0842584cd805b72fa2898c5bf8c674faf521/src/use-cases/tag-use-case.ts#L17)) or `string[]` (which is passed into the request body as described in the PR).

This means wrapping the input in an array will either evaluate to respectively:
1. `['["PSF", "SHO"]']` or
2. `[["PSF", "SHO"]]` (most likely)

In the case of (1), we should probably convert the string to an array using `split`. 
In the case of (2), we'll have to remove the additional brackets and pass `input.project` as is (TypeScript will complain, but we can fix that).

Reply:
Yes, this will not work.
We need to modify the type of the use case to accept multiple projects, so, string[] and then we can update the parent use case (CreateRelease) and the endpoint, (CreateReleaseEndpoint)

I am ok making projects and array and not a union with string. Makes things clearer